bayesian_cnn_prometheus/constants.py

```diff
@@ -0,0 +1,48 @@
+from pathlib import Path
+
+
+class DatasetType:
+    TRAIN = 'train'
+    TEST = 'test'
+    VALID = 'valid'
+
+
+class DatasetTypeTargets:
+    TRAIN = 'train_targets'
+    TEST = 'test_targets'
+    VALID = 'valid_targets'
+
+
+class Paths:
+    PROJECT_DIR = Path(__file__).parent
```

Comment:
Nie wiem czy chcemy się o to martwić, ale to się zepsuje jak przeniesiemy plik do innego folderu.
Według mnie tutaj bardziej pasowało by 
```python
import bayesian_cnn_prometheus
Path(bayesian_cnn_prometheus).parent
```